### PR TITLE
Improve Readme file

### LINKED ISSUE #36: Some inperfection in the how-to
Hi dear contributor,

- You need to add `npm install` before `npm run serve`
- You must specify where add `include:componentDoc(height=55) button/button.md` in the file.
- Please add a introductory sentence before tikui.scss (I create 2 files but I edit another file just after. WTF in my mind) 

Thank for you help.
It's a nice project.

### PR DESCRIPTION
**Pull request by sbally**
Closes #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,13 +87,15 @@ And inside `button.code.pug`:
...
 
+### Style 
 Then you can see a button on the browser. Now, you have to change the appearance of this button on `_atom.scss` and `_button.scss` files:
 
 ```bash
 touch src/atom/_atom.scss
 touch src/atom/button/_button.scss
 ```
 
+Before going into these two new files, edit the default scss file:
```
Comment (Gnuk) [inline README.md]: Add an empty line after.

File: README.md [hunk 2]
```diff
@@ -9,10 +9,19 @@ Tikui is a MIT-licensed free software project allowing you to create a web patte
 
 ## Prerequisites
 
-* [Node.js](https://nodejs.org)
+* Install [Node.js](https://nodejs.org) LTS version
+* Install [npm](https://www.npmjs.com)
```
Comment (Gnuk) [inline README.md]: Npm is installed with nodejs, so you can remove this line.

File: README.md [hunk 3]
```diff
@@ -76,13 +87,15 @@ And inside `button.code.pug`:
 button.tikui-button Button
 ```
 
+### Style
```
Comment (Gnuk) [inline README.md]: Please add an empty line after this line

File: README.md [hunk 4]
```diff
@@ -9,10 +9,18 @@ Tikui is a MIT-licensed free software project allowing you to create a web patte
...
 ## Prerequisites
 
-* [Node.js](https://nodejs.org)
+* [Node.js](https://nodejs.org) LTS version
 
 ## Development
 
+### Install dependencies
+
+After clone the repository, go in the root tikui directory and launch this command:
```
Comment (jeremietisserand) [inline README.md]: `After the repository clone`

Comment (Gnuk) [reply README.md]: Or maybe `After cloning the repository, please go to the root Tikui directory and run this command:` ?

Comment (Gnuk) [inline README.md]: I think it's better to have `After cloning the repository, please go to the root Tikui directory and run this command:`.

Comment (sbally) [reply README.md]: done

Comment (sbally) [reply README.md]: done

**Pull request merged**